inventory/sample/group_vars/all/azure.yml

```diff
@@ -12,3 +12,4 @@
 # azure_vnet_name:
 # azure_vnet_resource_group:
 # azure_route_table_name:
+# azure_vmtype:
```

Comment:
please change to
```
# azure_vmtype: standard
```

Reply:
@LuckySB Do I need to change in the inventory/sample/group_vars/all/azure.yml file too? Because I have updated the docs/azure.md with the above comment.